### ISSUE TITLE
Incorrect 'aria-checked' attribute value

### ISSUE THREAD

**Issue by gatanaso** [vaadin/vaadin-checkbox#9]
### Description
Incorrect 'aria-checked' attribute value;

### Expected outcome
The 'aria-checked' attribute value should be 'mixed'.

### Actual outcome
The 'aria-checked' attribute value is 'checked'

<img width="816" alt="vaadin-checkbox-aria" src="https://user-images.githubusercontent.com/15094658/29453437-58559cf4-8412-11e7-9b2c-c7369e146c83.png">

### Live Demo
https://jsfiddle.net/goranvaadin/4sf4avtm/

### Steps to reproduce
1. Set the vaadin-checkbox 'indeterminate' state to 'true'
2. Set the vaadin-checkbox 'checked' state to 'true'

NOTE: If the steps are reversed, then the 'aria-checked' attribute has the correct value of 'mixed'

### Browsers Affected
- [x] Chrome
